client/src/components/controls/Controls.vue

```diff
@@ -62,11 +64,11 @@ export default defineComponent({
 <template>
   <div
     v-mousetrap="[
-      { bind: 'left', handler: mediaController.prevFrame, disabled: visible() },
-      { bind: 'right', handler: mediaController.nextFrame, disabled: visible() },
+      { bind: 'left', handler: throttle(mediaController.prevFrame, 10), disabled: visible() },
+      { bind: 'right', handler: throttle(mediaController.nextFrame, 10), disabled: visible() },
       { bind: 'space', handler: togglePlay, disabled: visible() },
-      { bind: 'f', handler: mediaController.nextFrame, disabled: visible() },
-      { bind: 'd', handler: mediaController.prevFrame, disabled: visible() },
+      { bind: 'f', handler: throttle(mediaController.nextFrame, 10), disabled: visible() },
+      { bind: 'd', handler: throttle(mediaController.prevFrame, 10), disabled: visible() },
```

Comment:
Yes, I added this because holding down right these keys eventually causes the page to freeze.  You're right that the location of the throttling should be `mediaController`.